src/utils/getUserEmail.ts

```diff
@@ -0,0 +1,19 @@
+import { getUserEmailFromColaJwt } from '@co-digital/login';
+import { log } from '../utils/logger';
+import { Request } from 'express';
+
+import * as config from '../config';
+
+export const getUserEmail = (req: Request) => {
```

Comment:
I would have changed it to

```suggestion
export const getUserEmail = (jwt: string): string => {
    const userEmail = getUserEmailFromColaJwt(jwt);

    if (!userEmail) {
        throw new Error('Failed to fetch the email.');
    }

    return userEmail;
};
```
Furthermore, we are going to use this method to fetch user's email and add it to the header when logging in.